src/expr.rs

```diff
@@ -1283,6 +1296,11 @@ fn rewrite_int_lit(
     shape: Shape,
 ) -> Option<String> {
     let symbol = token_lit.symbol.as_str();
+    let suffix = token_lit.suffix.as_ref().map(|s| s.as_str());
+
+    if suffix == Some("f32") || suffix == Some("f64") {
+        return rewrite_float_lit(context, token_lit, span, shape);
+    }
```

Comment:
Looks like the `token::Lit` has a nice method for this already 😁. Check out [is_semantic_float](https://doc.rust-lang.org/stable/nightly-rustc/rustc_ast/token/struct.Lit.html#method.is_semantic_float)

It would be simpler to write:

```suggestion
    if token_lit.is_semantic_float() {
        return rewrite_float_lit(context, token_lit, span, shape);
    }
```

Reply:
> Looks like the token::Lit has a nice method for this already 😁. Check out [is_semantic_float](https://doc.rust-lang.org/stable/nightly-rustc/rustc_ast/token/struct.Lit.html#method.is_semantic_float)

Oh, neat) Thanks!